biodatasets/diann_iber_eval/diann_iber_eval.py

```diff
@@ -0,0 +1,441 @@
+@proceedings{DBLP:conf/sepln/2018ibereval,
+  author    = {Paolo Rosso and
+               Julio Gonzalo and
+               Raquel Martinez and
+               Soto Montalvo and
+               Jorge Carrillo de Albornoz},
+  title     = {Proceedings of the Third Workshop on Evaluation of Human Language
+               Technologies for Iberian Languages (IberEval 2018) co-located with
+               34th Conference of the Spanish Society for Natural Language Processing
+               {(SEPLN} 2018), Sevilla, Spain, September 18th, 2018},
+  series    = {{CEUR} Workshop Proceedings},
+  publisher = {CEUR-WS.org},
+  volume    = {2150},
+  year      = {2018},
+  url       = {http://ceur-ws.org/Vol-2150},
+  biburl    = {https://dblp.org/rec/conf/sepln/2018ibereval.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DATASETNAME = "diann_iber_eval"
+
+_DESCRIPTION = """\
+DIANN's corpus consists of a collection of 500 abstracts from Elsevier journal papers
+related to the biomedical domain, where both Spanish and English versions are available.
+
+This dataset contains annotations for disabilities appearing in these abstracts,
+usually expressed either with a specific word, such as "blindness", or as the limitation
+or lack of a human function, such as "lack of vision".
+
+(Spanish)
+El corpus DIANN se compone de una colección de 500 resúmenes de artículos de revista Elsevier
+del ámbito biomédico, con versiones en español e inglés.
+
+Este conjunto de datos contiene anotaciones para discapacidades que aparecen en dichos resúmenes,
+expresadas por medio de palabras específicas, como "ablepsia", or como la limitación o falta de
+una función, como "falta de visión".
+"""
+
+_HOMEPAGE = "http://nlp.uned.es/diann/"
+
+_LICENSE = "UNKNOWN"
+
+_URLS = {
+    "diannibereval_en": {
+        "en": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/english.zip"
+    },
+    "diannibereval_es": {
+        "es": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/spanish.zip"
+    },
+    "diannibereval": {
+        "es": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/spanish.zip",
+        "en": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/english.zip",
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+_LANG2CODE = {"English": "en", "Spanish": "es"}
+_CODE2LANG = {"en": "English", "es": "Spanish"}
+
+
+class DIANNIberEvalDataset(datasets.GeneratorBasedBuilder):
+    """DIANN's corpus is a collection of 500 abstracts related to the biomedical domain, in Spanish and English"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = []
+    BUILDER_CONFIGS.append(
+        BigBioConfig(
+            name="diannibereval_source",
```

Comment:
can we make this consistent with dataset name please 
```suggestion
            name="diann_iber_eval_source",
```